Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -195,9 +195,7 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
 
             localEndpoint = localEndpoint.withTrack(trackId: videoTrack.rtcTrack().trackId, metadata: metadata)
 
-            if state == .connected {
```

Comment:
Why was this removed?

Reply:
Try to connect two ios devices with this line. They won't see each other's tracks.
It's because the `state` refers to peer connection state and not state of the app in general. It's necessary to send `renegotiate` message before peer connection is established in order to start the process of establishing peer connection.